Add drag and drop functionality to the page

Dor-sketch opened the pull request.

Dor-sketch committed:
Add drag and drop functionality to the page

stylesuxx commented on index.html:
```diff
@@ -15,82 +16,78 @@ <h1>Steganography Online</h1>
-            </div>
-          </form>
-          <div class="clearfix"></div>
+  <div class="tab-pane active" id="encode">
```
Please keep the indentation.

Dor-sketch replied:
Sorry for messing up the indentation. Thanks for the quick response and for building this tool. It assisted me in a CTF challenge. I've corrected the indentation.

stylesuxx reviewed (changes requested):
Thanks for your contribution!

I would just like to ask you to keep the indentation as it was, otherwise those changes are polluting your actual PR.

Dor-sketch committed:
Restored the indentation in the index.html file

Restored the indentation in the index.html file to avoid polluting the git history and pull requests with unnecessary changes.
